src/openzeppelin/account/IAccount.cairo

```diff
@@ -11,11 +11,18 @@ namespace IAccount {
     func supportsInterface(interfaceId: felt) -> (success: felt) {
     }
 
-    func isValidSignature(hash: felt, signature_len: felt, signature: felt*) -> (isValid: felt) {
+    func isValidSignature(
+        hash: felt,
+        signature_len: felt,
+        signature: felt*
+    ) -> (isValid: felt) {
```

Comment:
```suggestion
    func isValidSignature(
            hash: felt,
            signature_len: felt,
            signature: felt*
        ) -> (isValid: felt) {
```
No more double indents for implicits?

Reply:
we had that before? these are explicit though. i lean towards single indent but no strong opinions

Reply:
Whoops! Wrong example, but yeah. Check the pre-cairo v0.10.0 releases:
https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.3.2/src/openzeppelin/account/presets/Account.cairo#L16-L24

I think the double indent is a bit easier to read, but either way works